tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+import enum
+import os
+import pathlib
+import subprocess
+from typing import TYPE_CHECKING, Callable, Final, Union
+
+import pytest
+from pact import Consumer, Pact, Provider
+from pact.matchers import Matcher  # noqa: TCH002
+from typing_extensions import Annotated  # noqa: TCH002
+
+from great_expectations.compatibility import pydantic
+from great_expectations.core.http import create_session
+
+if TYPE_CHECKING:
+    from requests import Session
+
+
+PACT_MOCK_HOST: Final[str] = "localhost"
+PACT_MOCK_PORT: Final[int] = 9292
+PACT_DIR: Final[str] = str(
+    pathlib.Path(pathlib.Path(__file__).parent, "pacts").resolve()
+)
+
+
+@pytest.fixture
+def existing_organization_id() -> str:
+    try:
+        return os.environ["GX_CLOUD_ORGANIZATION_ID"]
+    except KeyError as e:
+        raise OSError("GX_CLOUD_ORGANIZATION_ID is not set in this environment.") from e
+
+
+class RequestMethods(str, enum.Enum):
+    DELETE = "DELETE"
+    GET = "GET"
+    PATCH = "PATCH"
+    POST = "POST"
+    PUT = "PUT"
+
+
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(scope="module")
```

We shouldn't create a new HTTP session for every test,

Reply:
Makes sense to me ✅